src/de/impls.rs

```diff
@@ -407,17 +436,17 @@ impl<'a, 'de: 'a> BorrowDecode<'de> for &'a [u8] {
     }
 }
 
-impl<'a, 'de: 'a> BorrowDecode<'de> for Option<&'a [u8]> {
-    fn borrow_decode<D: BorrowDecoder<'de>>(decoder: &mut D) -> Result<Self, DecodeError> {
-        match super::decode_option_variant(decoder, core::any::type_name::<Option<&[u8]>>())? {
-            Some(_) => {
-                let val = BorrowDecode::borrow_decode(decoder)?;
-                Ok(Some(val))
-            }
-            None => Ok(None),
-        }
-    }
-}
+// impl<'a, 'de: 'a> BorrowDecode<'de> for Option<&'a [u8]> {
```

Comment:
You have left commented out code in the commit. Burn it :fire:

Reply:
I added a BlockedTODO for specialization, which is why I wanted to keep this code